app/src/main/java/com/example/vmedvediev/ua21summerdancecamp/UI/bottomnavigation/events/EventsViewModel.kt

```diff
@@ -4,25 +4,30 @@ import android.arch.lifecycle.LiveData
 import android.arch.lifecycle.MutableLiveData
 import android.arch.lifecycle.ViewModel
 import android.util.Log
+import com.example.vmedvediev.ua21summerdancecamp.model.Date
 import com.example.vmedvediev.ua21summerdancecamp.model.Event
+import com.example.vmedvediev.ua21summerdancecamp.model.ListItem
+import com.example.vmedvediev.ua21summerdancecamp.model.RealmEvent
 import com.example.vmedvediev.ua21summerdancecamp.repository.Repository
 
 class EventsViewModel(private val repository: Repository) : ViewModel() {
 
-    private val events: MutableLiveData<ArrayList<Event>> = MutableLiveData()
+    private val events: MutableLiveData<ArrayList<ListItem>> = MutableLiveData()
 
     fun getDataBydDate(date: String) {
-        repository.getEventsList(date, {eventsList: ArrayList<Event> -> onDataLoaded(eventsList)},
-                {eventsList: ArrayList<Event> -> onDataNotLoaded(eventsList)})
+        repository.getEventsList(date, {eventsList: ArrayList<ListItem> -> onDataLoaded(eventsList)},
+                {eventsList: ArrayList<ListItem> -> onDataNotLoaded(eventsList)})
     }
 
-    fun getEventsList() : LiveData<ArrayList<Event>> = events
+    fun getEventsLiveData() : MutableLiveData<ArrayList<ListItem>> = events
 
-    private fun onDataLoaded(eventsList: ArrayList<Event>) {
+    fun getEventsList() : ArrayList<ListItem> = events.value!!
+
+    private fun onDataLoaded(eventsList: ArrayList<ListItem>) {
```

Comment:
Don't you think that these two methods are identical? :)
So, probably, you need just one method...